Eatery/Controllers/Eateries/Campus/ScrollableViewController.swift

```diff
@@ -0,0 +1,374 @@
+//
+//  ScrollableViewController.swift
+//  Eatery
+//
+//  Created by Sergio Diaz on 12/9/20.
+//  Copyright © 2020 Cornell AppDev. All rights reserved.
+//
+
+import UIKit
+
+class ScrollableViewController: UIViewController {
+
+    private var eatery: CampusEatery!
+    private var viewControllers: [CampusEateryExpandedMenuViewController]!
+    private var allItems: [ExpandedMenu.Item]!
```

Comment:
Can we make these into `private let`'s

Reply:
It might also be good to add a few doc comments to these properties. Is `allItems` supposed to be in sorted order?